browser/brave_wallet/brave_wallet_importer_delegate_impl.cc

```diff
@@ -0,0 +1,135 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_wallet/brave_wallet_importer_delegate_impl.h"
+
+#include <utility>
+
+#include "base/bind_post_task.h"
+#include "base/threading/sequenced_task_runner_handle.h"
+#include "brave/browser/ethereum_remote_client/ethereum_remote_client_constants.h"
+#include "brave/browser/ethereum_remote_client/ethereum_remote_client_service.h"
+#include "brave/browser/ethereum_remote_client/ethereum_remote_client_service_factory.h"
```

Comment:
Put these behind a buildflag too pls

Reply:
fixed in 615ec530a09034306bb5efdea47492e0464e5558